src/tests/system/pytest.ini

```diff
@@ -2,6 +2,13 @@
 addopts = --strict-markers
 testpaths = tests
 markers =
-    slow: marks tests as slow (deselect with '-m "not slow"')
+    authentication: marks authentication test
+    authorization: marks authorization test
+    cache: marks any cache related test
     contains_workaround_for(gh=...,bz=...): test requires workaround for an existing bug
+    identity: marks identity, lookups test
```

Comment:
Unused and probably irrelevant since tests that should be marked with will most likely only reside in test_id.py

Reply:
I see that short names, id mapping, and user lookups from other domains will also fall under this category, reconsider?

Reply:
Ok.